node/npm_dependencies.ts

```diff
@@ -14,6 +15,51 @@ import { Logger } from './logger.js'
 
 const TYPESCRIPT_EXTENSIONS = new Set(['.ts', '.cts', '.mts'])
 
+/**
+ * Returns the name of the `@types/` package used by a given specifier. Most of
+ * the times this is just the specifier itself, but scoped packages suffer a
+ * transformation (e.g. `@netlify/functions` -> `@types/netlify__functions`).
+ * https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages
+ */
+const inferDefinitelyTypedPackage = (specifier: string) => {
```

Comment:
nit: I would call this `getTypesPackageName`. "Definitely typed" doesn't mean a lot to people who don't know that the repo where the types live have that name, so I think it could be a bit more descriptive.

Also you're calling this "types package" below, so it would make things a bit more consistent.

Reply:
renamed in https://github.com/netlify/edge-bundler/pull/505/commits/a0a27f0cc7a1beee318a17642450fefe09afcb66